plugins/python-build/bin/python-build

```diff
@@ -806,6 +806,9 @@ build_package_standard_build() {
       use_homebrew_zlib || true
     fi
     use_dsymutil || true
+    if ! is_mac; then
+      build_package_enable_shared || true
+    fi
```

Comment:
What's the rationale for "non-Mac"?

I only found that it's incompatible with `--enable-framwework` so we can probably only not apply it if that has been passed.

Reply:
https://github.com/python/cpython/blob/cd67c1bb30eccd0c6fd1386405df225aed4c91a9/configure.ac#L3049